src/llvm-simdloop.cpp

```diff
@@ -156,41 +159,18 @@ bool LowerSIMDLoop::markLoopInfo(Module &M, Function *marker)
         if (!L)
             continue;
 
-        DEBUG(dbgs() << "LSL: loopinfo marker found\n");
-        bool simd = false;
-        bool ivdep = false;
-        SmallVector<Metadata *, 8> MDs;
-
+        LLVM_DEBUG(dbgs() << "LSL: simd_loop found\n");
+        LLVM_DEBUG(dbgs() << "LSL: ivdep is: " << ivdep << "\n");
         BasicBlock *Lh = L->getHeader();
-        DEBUG(dbgs() << "LSL: loop header: " << *Lh << "\n");
-
-        // Reserve first location for self reference to the LoopID metadata node.
-        TempMDTuple TempNode = MDNode::getTemporary(Lh->getContext(), None);
-        MDs.push_back(TempNode.get());
-
-        // Walk `julia.loopinfo` metadata and filter out `julia.simdloop` and `julia.ivdep`
-        if (I->hasMetadataOtherThanDebugLoc()) {
-            MDNode *JLMD= I->getMetadata("julia.loopinfo");
-            if (JLMD) {
-                DEBUG(dbgs() << "LSL: has julia.loopinfo metadata with " << JLMD->getNumOperands() <<" operands\n");
-                for (unsigned i = 0, ie = JLMD->getNumOperands(); i < ie; ++i) {
-                    Metadata *Op = JLMD->getOperand(i);
-                    const MDString *S = dyn_cast<MDString>(Op);
-                    if (S) {
-                        DEBUG(dbgs() << "LSL: found " << S->getString() << "\n");
-                        if (S->getString().startswith("julia")) {
-                            if (S->getString().equals("julia.simdloop"))
-                                simd = true;
-                            if (S->getString().equals("julia.ivdep"))
-                                ivdep = true;
-                            continue;
-                        }
-                    }
-                    MDs.push_back(Op);
-                }
-            }
```

Comment:
merge/rebase conflict? or was this not used?

Reply:
That's indeed a mistake, fixing in next commit.